formulae/matrices.py

```diff
@@ -522,7 +522,13 @@ def design_matrices(formula, data, na_action="drop", env=0):
     incomplete_rows_n = incomplete_rows.sum()
 
     if incomplete_rows_n > 0:
-        if na_action == "drop":
+        if na_action == "pass":
+            _log.info(
+                "Keeping %s/%s rows with na in the dataset.",
```

Comment:
I think that 

```python
"Keeping %s/%s rows with at least one missing value in the dataset."
```

is more accurate. Can we change that?

Reply:
done